README.md

```diff
@@ -32,7 +32,9 @@ and assumes the environment variables that the activation script sets):
 
     source ./env/bin/activate
 
-After having created the python virtual environment and having activated it the package
+To build the project you will also need node.js. Please read the instructions for your system [here](https://nodejs.org/en/download/package-manager/).
```

Comment:
Forget previous remark, we need Node.js. We use it to have npm but the scripts that precompiles the templates needs Node.js on the system anyway.